### PR TITLE
[NFR] - added description property to Cli\Router\Route

### PR DESCRIPTION
**Pull request by ekmst**
Hello!

* Type: new feature
* Link to issue:

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [x] I updated the CHANGELOG

Small description of change: added description property to Cli\Router\Route

## Usage: 
```php
$router = new \Phalcon\Cli\Router(false);
$router->add('test')->setName('testName')->setDescription('testDescription');
$router->add('test1')->setName('testName1')->setDescription('testDescription1');
$router->add('test2')->setName('testName2')->setDescription('testDescription2');

foreach ($router->getRoutes() as $route) {
    printf("%s    %s\n", $route->getName(), $route->getDescription());
}
```

```
$./phalconcli
Available command or tasks:
   testName      testDescription
   testName1     testDescription1
   testName2     testDescription2
```
Thanks

### CODE REVIEW

File: tests/cli/Cli/Router/Route/GetDescriptionCest.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+     * @param CliTester $I
+     *
+     * @since 2019-04-02
+     */
+    public function cliRouterRouteGetDescription(CliTester $I)
+    {
+        $I->wantToTest('Phalcon\Cli\Router\Route - getDescription()');
+
+        $routes = [
+            'test' => 'test description',
```
Comment (Jeckerson) [inline tests/cli/Cli/Router/Route/GetDescriptionCest.php]: Make sense to move that array to dataProvider method.

File: tests/cli/Cli/Router/Route/SetDescriptionCest.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+     * @param CliTester $I
+     *
+     * @since 2019-04-02
+     */
+    public function cliRouterRouteSetDescription(CliTester $I)
+    {
+        $I->wantToTest('Phalcon\Cli\Router\Route - setDescription()');
+
+        $routes = [
+            'test' => 'test description',
```
Comment (Jeckerson) [inline tests/cli/Cli/Router/Route/SetDescriptionCest.php]: Make sense to move that array to dataProvider method.

**Comment by ekmst** [comment]
@Jeckerson done

**Review by sergeyklay — DISMISSED** [review]

**Review by Jeckerson — DISMISSED** [review]

File: phalcon/Cli/Router/RouteInterface.zep [hunk 3]
```diff
@@ -72,8 +77,13 @@ interface RouteInterface
...
+    /**
+     * Sets the route's description
+     */
+    public function setDescription(string! description) -> <Route>;
+
     /**
      * Sets the route's name
      */
-    public function setName(string name) -> void;
+    public function setName(string name) -> <Route>;
```
Comment (niden) [inline phalcon/Cli/Router/RouteInterface.zep]: Shouldn't this be `<RouteInterface>`?

File: phalcon/Cli/Router/Route.zep [hunk 4]
```diff
@@ -531,6 +541,15 @@ class Route
         let self::uniqueId = null;
     }
 
+    /**
+     * Sets the route's description
+     */
+    public function setDescription(string! description) -> <Route>
```
Comment (niden) [inline phalcon/Cli/Router/Route.zep]: Same here these need to be `RouteInterface` (for `setName()` also)

**Review by niden — CHANGES_REQUESTED** [review]

**Review by sergeyklay — APPROVED** [review]

**Review by Jeckerson — APPROVED** [review]

**Comment by niden** [comment]
Thank you @ekmst

**Pull request merged**